packaging/kubernetes-deployment/sample-ebpfprograms/gocounter.yaml

```diff
@@ -0,0 +1,91 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: gocounter-config
+  namespace: kube-system
+data:
+  gocounter.toml: |
+    [tls]
+      ca_cert = "/etc/bpfd/certs/ca/ca.crt"
+      cert = "/etc/bpfctl/certs/gocounter/tls.crt"
+      key = "/etc/bpfctl/certs/gocounter/tls.key"
+    [config]
+      interface = "ens3"
+      bytecode_uuid = "62c2a443-9beb-4261-9c46-33a29612643d"
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: gocounter-cert
+  namespace: kube-system
+spec:
+  dnsNames:
+    - localhost
+  ipAddresses:
+    - 127.0.0.1
+  commonName: gocounter
+  secretName: gocounter-cert-secret
+  privateKey:
+    algorithm: RSA
+  issuerRef:
+    name: bpfd-cert-issuer
+    kind: Issuer
+    group: cert-manager.io
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: gocounter-pod
+spec:
+  hostNetwork: true
+  dnsPolicy: ClusterFirstWithHostNet
+  terminationGracePeriodSeconds: 15
+  nodeSelector:
+    kubernetes.io/hostname: ebpf-k8s-2
+  containers:
+  - name: gocounter
+    image: quay.io/$USER/gocounter:latest
+    imagePullPolicy: IfNotPresent
+    #securityContext:
```

Comment:
I couldn't get it to work manually because docker doesn't know about CAP_BPF. I was tracing the patch but it wasn't in my version. It should work this way once docker knows about CAP_BPF. Need to test on K8s but got side tracked. So left by accident but should be done this way long term.

Reply:
Ok cool maybe just add an issue to investigate this/ perms in K8s in general and remove this for now?

Reply:
Removed for now.